bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -103,6 +103,26 @@
 
 	</thing-type>
 
+	<thing-type id="window-contact-2">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Door/Window Contact II</label>
+		<description>Detects open windows and doors and features an additional button.</description>
+
+		<channels>
+			<channel id="contact" typeId="contact"/>
+			<channel id="battery-level" typeId="system.battery-level"/>
+			<channel id="low-battery" typeId="system.low-battery"/>
+			<channel id="bypass-state" typeId="bypass-state"/>
+			<channel id="communication-quality" typeId="system.signal-strength"/>
```

Comment:
I thought about that as well. In the Bosch world the service name is CommunicationQuality, in the openHAB world it's signal strength. So one could argue for both approaches. If other bindings favor the openHAB terminology in most cases, we should probably do so as well for the sake of consistency.

Reply:
That's the general idea with openHAB: "a vendor and technology agnostic open source automation software for your home". 🙂

If Bosch Smart Home users are already familiar with the term "communication quality" (from a Bosch app for example), I think it would be a nice addition to override the channel type description to mention this (perhaps also in the README), but if this is only an API term, I don't think it's needed, since it shouldn't affect users.

The benefit of using openHAB terminology consistently is that you can easily replace a device with a device from another vendor (even eco-system), and this will be fully abstracted. Items can be relinked to the new Thing, and rules, UI's, etc. will be unaffected. If the terminology is also the same, it will be easier to understand how to correctly relink/reuse.

In this case it's only the channel id - the label/description is already taken from the system channel type, so I think it's just a matter of consistency.